bulk_grades/migrations/0003_auto_20210101_0652.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.2.17 on 2021-01-01 06:52
```

Comment:
Could we use a more descriptive filename, maybe `0003_drop_djcelery_tables`?  I really wish we had some kind of linter to catch that, the `auto_*` filenames make it tough to scan the migrations and see what changed over time.